flow/src/main/java/com/sequenceiq/flow/service/FlowProgressService.java

```diff
@@ -0,0 +1,50 @@
+package com.sequenceiq.flow.service;
+
+import static com.google.common.base.Preconditions.checkState;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.auth.crn.Crn;
+import com.sequenceiq.cloudbreak.common.exception.NotFoundException;
+import com.sequenceiq.flow.api.model.FlowProgressResponse;
+import com.sequenceiq.flow.converter.FlowProgressResponseConverter;
+import com.sequenceiq.flow.domain.FlowLog;
+import com.sequenceiq.flow.service.flowlog.FlowLogDBService;
+
+@Service
+public class FlowProgressService {
```

Comment:
some unit test would be nice :)

Reply:
The whole flow progress thing will be purged, and it is just a copy-paste to avoid circular dependency